package.json

```diff
@@ -19,10 +19,14 @@
     "deploy-testnet": "graph deploy --product hosted-service --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ lidofinance/lido-testnet --network goerli",
     "create-local": "env-cmd --use-shell graph create --node '$GRAPH_NODE' lidofinance/lido",
     "remove-local": "env-cmd --use-shell graph remove --node '$GRAPH_NODE' lidofinance/lido",
-    "deploy-local": "env-cmd --use-shell graph deploy --node '$GRAPH_NODE' --ipfs '$GRAPH_IPFS' lidofinance/lido --network mainnet -l v2.0.0",
+    "deploy-local": "env-cmd --use-shell graph deploy --node '$GRAPH_NODE' --ipfs '$GRAPH_IPFS' lidofinance/lido --network mainnet -l v2.0.1",
```

Comment:
Do we need to hardcode a version here? From now on (after v2 release), that's probably not necessary

Reply:
Same for `deploy-local-testnet`

Reply:
Same question for other commands